app/models/manageiq/providers/redhat/infra_manager/ovirt_services.rb

```diff
@@ -352,7 +352,8 @@ def advertised_images
 
         sd_service = ems_service.system_service.storage_domains_service.storage_domain_service(iso_sd.id)
         iso_images = sd_service.files_service.list
-        iso_images.collect(&:name)
+
+        iso_images
```

Comment:
Since `OvirtServices` is a simple class we should be able to simply inherit from `Ovirt::InfraManager::OvirtServices` without duplicating the code.  Don't want to hold this up but we can probably delete all this code in a follow-up

Reply:
Thanks, I'll take note for a future follow-up PR